ch5/ex-5-13-tail.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Exercise 5-13
+ *
+ * Write the program tail, which prints the last n lines of its input.  By
+ * default, n is 10, let us say, but it can be changed by an optional
+ * argument, so that
+ *
+ * 	tail -n
+ *
+ * prints the last n lines.  The program should behave rationally no matter
+ * how unreasonable the input or the value of n.  Write the program so it
+ * makes the best use of available storage; lines should be stored as in the
+ * sorting program of Section 5.6, not in a two-dimensional array of fixed
+ * size.
+ */
+
```

Comment:
Pieter says stop now and use C++

Reply:
It looks like any random idiot can turn up and post mean things about your code! How troublesome.